sdk/eventhub/Azure.Messaging.EventHubs/tests/Primitives/EventProcessorTests.PartitionProcessing.cs

```diff
@@ -180,6 +180,7 @@ public void CreatePartitionProcessorPreservesTheCancellationSource()
         /// </summary>
         ///
         [Test]
+        [Ignore("Intermittent failures in CI; investigation required.")]
```

Comment:
Can we link to tracking issues in this?

Reply:
We could but I think that I've already fixed it in another branch, so I'm not sure there's a ton of value.   I'm planning on closing out the issue when the ignore comes through.